data/portchain.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Portchain",
+  "career_page_url": "https://www.portchain.com/international-team/",
+  "url": "https://www.portchain.com/",
+  "remote_policy": "Optional",
+  "hiring_policy": "Contract",
+  "type": "Product",
+  "category": "cloud_software",
```

Hi @fgandellini, first of all thank you for your contribution!
```suggestion
  "categories": [ 
    "cloud_software",
    ]
```

We recently switched to a multi-category system, can you please fix your PR accordingly? You can find more info [here](https://github.com/italiaremote/awesome-italia-remote/blob/main/CONTRIBUTING.md). Thanks

---

Sure! Thanks for the blazing fast reply!